.github/blog-template-demo.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: ''
+Author: Your name here
+Author handle: Any links you want associated with your name
```

Comment:
We should couple any additional front matter parameters here with a change to the blog template on the website:

https://github.com/knative/website/blob/master/layouts/blog/baseof.html

Reply:
I think we're also mostly using the Docsy Blog template here: https://github.com/google/docsy/blob/37503b4b16381071e1941abe6079ce09837db3f2/layouts/blog/content.html#L5

Note that the referenced template only has `.Params.author`, but allows you to include markdown in the "author" info.